test/compress/functions.js

```diff
@@ -0,0 +1,6 @@
+non_ascii_function_identifier_name: {
+    input: {
+        function fooλ(δλ) {}
```

Comment:
Please a add a test with function name starting with a non-ascii character - might as well make it a function expression while we're at it to test AST_Function vs. AST_Defun: 
```
(function λ(δλ) {})();
```